core/kazoo_csv/download_csvs.py

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env python2
-
-import requests, sys, json
-
-print("version: ", requests.__version__)
-csvs_url = 'https://api.github.com/repos/maxogden/csv-spectrum/contents/csvs'
```

Comment:
Yes, attribution is important. Why force people to dig through the git history to find out where the files came from. We didn't create them; they did.

If CI isn't running them, CI should be updated to run them. If they don't pass, that's a regression cause tests passed when I added them